lib/dsl/poc/ast_builder.js

```diff
@@ -19,10 +19,12 @@ const BaseJDLCSTVisitor = new JDLParser().getBaseCstVisitorConstructor();
  * 1. ctx.NAME[0].image --> ctx.NAME[0] ? ctx.NAME[0].image : "???"
  */
 class JDLAstBuilderVisitor extends BaseJDLCSTVisitor {
-  constructor() {
-    super();
-    this.validateVisitor();
-  }
+  // constructor() {
```

Comment:
Why is it not needed anymore?

Reply:
It is needed, I think I will re-add it and add the missing methods.